src/ApiRequestPanel.js

```diff
@@ -474,6 +495,7 @@ export class ApiRequestPanel extends AmfHelperMixin(EventsTargetMixin(HeadersPar
     this.addEventListener('api-request', this._apiRequestHandler);
     this.addEventListener('api-server-changed', this._serverChangeHandler);
     this.addEventListener('api-navigation-selection-changed', this._handleNavigationChange);
+    this.addEventListener('api-servers-count-changed', this._handleServersCountChange);
```

Comment:
Where this event is dispatched? Does it bubble? I don't think this should be a bubbling event. It should ber only relevant to the parent hosting element. In this case you handle the event directly on the element template:
```javascript
<api-server-selector
 @api-servers-count-changed="${this._handleServersCountChange}"
></api-server-selector>
```
Then there's no need to bind the handler function.